src/js/angular-tooltips.js

```diff
@@ -26,7 +26,7 @@
           , width
           , offsetTop
           , offsetLeft
-          , title = attr.title || ''
+          , title = attr.tooltipTitle || ''
```

Comment:
Ok so better as i shown ;)

Just use attr.tooltipTitle || attr.title ... so that if tooltipTitle isn't available will inherit from title tag :100: